jupyter_core/command.py

```diff
@@ -113,9 +119,29 @@ def _execvp(cmd, argv):
         os.execvp(cmd, argv)
 
 
+def _jupyter_abspath(path, subcommand):
+    """This method get the abspath of jupyter with no changes on ENV."""
+    # get env PATH with self
+    search_path = os.pathsep.join(_path_with_self())
+    # get the abs path for the jupyter-<subcommand>
+    jupyter_subcommand = '{}-{}'.format(path, subcommand)
```

Comment:
Thanks. Now you don't need to pass an argument, you can just hardcode it in this line. That's one less thing to follow around when reading the code. :-)

Reply:
Ok :) done!